src/xrpld/rpc/handlers/Submit.cpp

```diff
@@ -83,11 +83,11 @@ doSubmit(RPC::JsonContext& context)
 
     SerialIter sitTrans(makeSlice(*ret));
 
-    std::shared_ptr<STTx const> stpTrans;
+    std::shared_ptr<STTx const> stTx;
```

Comment:
Please avoid Hungarian notation

Reply:
This isn't hungarian notation, just `STTx` in `camelCase`.